docs/commands/completion.md

```diff
@@ -14,13 +14,15 @@ The output is a script for you to copy and paste, and add to the configuration f
 
```

Comment:
hmm, for some reason this file is not included in the sidebar navigation menu. I will need to check with team to find out why. @Redocly/dark-side

Reply:
It's not exactly a command, it's more like an extended install step. You wouldn't use this by itself so I linked it from locations where it's useful, but it's not a command as such.